blog/migrations/0004_create_superuser.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.contrib.auth.hashers import make_password
+
+def create_superuser(apps, schema_editor):
+    User = apps.get_model('auth', 'User')
+    User.objects.create(username='chronokeeper',
```

Comment:
But in general I strongly **do not recommend** mixing different changes in one PR/commit.